scout/parse/variant/variant.py

```diff
@@ -280,7 +282,7 @@ def parse_variant(variant, case, variant_type='clinical',
         parsed_variant['spidex'] = float(spidex)
 
     ###################### Add conservation ######################
-    parsed_variant['conservation'] = parse_conservations(variant, vep_header)
+    parsed_variant['conservation'] = parse_conservations(variant, vep_conservation, raw_transcripts)
```

Comment:
Unfortunately this will not work since `raw_transcripts` is a generator with will be exhausted in line 225. One idea could be to parse the information in the `parse_transcripts` function. I guess it is easier if we sit down together tomorrow

Reply:
Mmm, What if we convert raw_transcripts into a list?

Reply:
But yes, tomorrow!

Reply:
Made it into a list, see you tomorrow! :)